assets/_scss/elements/_inputs.scss

```diff
@@ -78,6 +81,17 @@ select {
   background-position: right 1.3rem center;
 }
 
+legend {
```

Comment:
@msecret I'm pretty sure legends can be used without a fieldset. It's just that if you use a fieldset, you must use a legend.